src/Temperature-Accretech_UFseries/libs/accretech_uf.py

```diff
@@ -275,7 +279,19 @@ def acquire_status_byte(self, timeout: float = 10.0) -> int:
                 print("-->", now.strftime("%H:%M:%S"), "Function:", function_calling_name)
 
         # Wait for the SRQ event to occur
-        response = self.port.port.wait_on_event(self.event_type, int(timeout * 1000))  # conversion from s to ms
+        starttime = time.time()
+        while True: 
+            if time.time()-starttime < timeout:
+                try:
+                    # response = self.port.port.wait_on_event(self.event_type, int(timeout * 1000))  # conversion from s to ms
```

Comment:
Seems this line is no longer needed.
But I'm curious, why was it necessary to introduce a while loop?

Reply:
The while-loop is needed to be able to stop the waiting for the event in future. It is not yet implemented, but a Stop in the GUI of SweepMe! could also stop the waiting. So far one needs to wait until the end and some processes have a timeout of 300 s to make sure all wafers are loaded.

Reply:
added a comment